setup.py

```diff
@@ -79,6 +79,7 @@
         'bitcoin==1.1.42',
         'flask==0.10.1',
         'requests==2.9',
+        'gunicorn',
```

Comment:
Gunicorn it's quite stable, and I don't think there will be many API changes in the future. In general, I'm reluctant in pinning the version in the `setup.py`, but I'm still not 100% sure how to handle it.

In general, if the project supports semantic versioning, we can stick with the major version. Should I pin the version to `19` then?